tls/ca_watcher.go

```diff
@@ -15,7 +15,8 @@ import (
 	"github.com/go-kit/log/level"
 )
 
-type CAWatcher struct {
+// caWatcher poll for changes on the CA file, if the CA change it will add it to the certificate Pool.
```

Comment:
What is the CA file? Is it the CA key, or the CA cert? 🤔 

I believe it's the certificate given what I see, but would be nice to be explicit about it.

Reply:
Done, I put the clarification